### PR TITLE
update setup

### PR DESCRIPTION
**Pull request by pkern90**

### CODE REVIEW

File: docker-compose.env [hunk 1]
```diff
@@ -1,12 +1,15 @@
-
-REDACT_PIPELINE_IMAGE="docker.brighter.ai/redact:v3.3.1-237"
-REDACT_INFER_IMAGE="docker.brighter.ai/redact-gpu:0.3.0"
+REDACT_IMAGE="brighter.azurecr.io/redact:v4.4.0-252"
```
Comment (seanconeys) [inline docker-compose.env:1]: wrong repo should all come from docker.brighter.ai

Comment (seanconeys) [reply docker-compose.env:1]: this is currently our internal repo

File: docker-compose.env [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+REDACT_GPU_IMAGE="brighter.azurecr.io/redact-infer:1.2.0"
 REDACT_UTILS_IMAGE="docker.brighter.ai/redact-utils:0.12.0"
 
+REDACT_CONTAINER_NAME="redact-debug"
```
Comment (seanconeys) [inline docker-compose.env:5]: remove debug from container names

File: start_redact.sh [hunk 3]
```diff
@@ -5,17 +5,47 @@ set -a
...
+container_name="${REDACT_CONTAINER_NAME} ${REDACT_GPU_CONTAINER_NAME}"
+if [ ${ui+x} ]; then
+    container_name="${container_name} ${REDACT_UTILS_CONTAINER_NAME}";
+fi
+
+# make sure there aren't any containers with the same names yet
+for container in ${container_name}; do
+    if [ $(docker ps -a -q -f name="^${container}$") ]; then
+        echo "A Container with the name '${container}' already exists please make sure to remove it before starting a new one."
+        echo "If you intent to start a seperate instance please choose a differnt container name in the env file."
```
Comment (seanconeys) [inline start_redact.sh:31]: intend not intent, different

File: docker-compose.env (hunk 1 above)
Comment (pkern90) [reply docker-compose.env:1]: done

File: start_redact.sh (hunk 3 above)
Comment (pkern90) [reply start_redact.sh]: done

**Pull request merged**